### ISSUE TITLE
Iota Light wallet 2.5.7 Balance is 0

### ISSUE THREAD

**Issue by paradon8** [iotaledger/wallet#1165]
### Prerequisites
If you are **not** suggesting a feature, you must be able to check all of the following (place an x inside the brackets to check the box). If you cannot check all of the boxes, this issue should not be reported on GitHub as it is not a bug:

* [ x] I confirm that this is an issue with the IOTA Wallet and not an exchange
* [ x] I confirm that this is an issue with the wallet, not IRI
* [ x] For pending transactions: I confirm that I have tried reattaching and promoting multiple times (check this box if your issue is not related to a pending transaction)
* [x ] I confirm that this is not an issue related to stolen/lost funds
* [x ] I confirm that this is not a zero balance issue related to a snapshot (last snapshots occurred on October 23 and January 28)
* [ x] For private key warnings: I confirm that I am not at risk of reusing an address, or another user has verified that I am not (check this box if your issue is not related to a private key reuse)
* [x ] I confirm that this issue is present on the latest version of the wallet

Hello everybody,

I have also the issue that my balance shows 0 and I'm really desperate right now.

I created a light wallet at 14/02/18 and genereated a seed (without online seed generator) and 1-3 adresses I guess. I logged out and logged in again with my seed (Unfortunately I didnt't check my checksum).
Afterwards I have sent my Miotas from binance to this adress:
CZO9FHKIXBPRKQMOUYDYKRVSAVUAHEYYHQZNWEXQGANDQLT9SHVENSFKSSEOUITAUK9DZSXNVCQREHGZWJNOBRYRRA

And the they also appeared in my wallet, so I logged out

In the last days I tried to login to my wallet with my seed. But the only address that was in the history was no the one mentioned above and the balance is 0. But as you can see the Miotas are still at the address where I have sent them to.

What I have already done now:
I generated around 30 addresses with different nodes and attached them to the tangle. Do I have to create more?
I did read and try all the steps in your guide

What can I do now? I'm really really desperate and I would aprreciate your help so much

Best regards,
Patrick

[OS Version]
[Wallet Version] Light Wallet 2.5.7

**Comment by wonkytonky** [comment]
try changing nodes, 

![image](https://user-images.githubusercontent.com/19390834/36935253-e7af711c-1ef5-11e8-8371-6232db66b3d9.png)

**Comment by paradon8** [comment]
I tried multiple nodes without success. But I noticed that the node that I used initially, doesn't work anymore "http://service.iotasupport.com:14265". it shows me "Invalid response" when I try to create addresses.

Since I copy pasted most of the code and just typed a few characters manually, I don't really know where the typo should be. 

Can you send me your java code with a little instructions? Maybe it helps me. Thank you already for your help.

**Comment by wonkytonky** [comment]
https://www.reddit.com/r/Iota/comments/6w4iny/iota_offline_address_generator_for/

replace with the lates iota.js  and follow instructions

be sure to ONLY use this on an offline pc that never touches the internet. 
if you cannot use this in an offline environment don't use it

**Comment by paradon8** [comment]
Thank you. As I understood, this is a tool to create seed and addresses offline, right? Or what does it actually do? Sorry for if I'm wrong.

But my problem is that I have generated an address, sent my miotas there and can't access it anymore...

**Comment by wonkytonky** [comment]
yes.. you can change  index nummer.. to see what address is on what index.. you start with 0 and go up ....  
this way you can check if your seed is wrong or not..   if your seed is correct .. at some point you should get the right address...    since you didnt created much addresses before you send money to your wallet this should not take long.....

**Comment by paradon8** [comment]
ALright, I will check that out.
But if my seed is wrong, the tool won't help me right?

**Comment by wonkytonky** [comment]
if  seed is wrong you cannot access the address indeed. nobody can

**Comment by CaliBitcoins** [comment]
Read this. I lost all my money too, but not from seed gen or phishing

https://medium.com/@neha/cryptographic-vulnerabilities-in-iota-9a6a9ddc4367

**Comment by SupaHam** [comment]
See #1187
